2021/24xxx/CVE-2021-24153.json

```diff
@@ -1,18 +1,85 @@
 {
-    "data_type": "CVE",
-    "data_format": "MITRE",
-    "data_version": "4.0",
-    "CVE_data_meta": {
-        "ID": "CVE-2021-24153",
-        "ASSIGNER": "cve@mitre.org",
-        "STATE": "RESERVED"
-    },
-    "description": {
-        "description_data": [
-            {
-                "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
-            }
+  "CVE_data_meta": {
+    "ID": "CVE-2021-24153",
+    "ASSIGNER": "contact@wpscan.com",
+    "STATE": "PUBLIC",
+    "TITLE": "Yoast SEO <= 3.4.0 - Authenticated Stored Cross-Site Scripting (XSS)"
+  },
+  "data_format": "MITRE",
+  "data_type": "CVE",
+  "data_version": "4.0",
+  "generator": "WPScan CVE Generator",
+  "affects": {
+    "vendor": {
+      "vendor_data": [
+        {
+          "vendor_name": "Unknown",
+          "product": {
+            "product_data": [
+              {
+                "product_name": "Yoast SEO",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_name": "3.4.1",
+                      "version_value": "3.4.1"
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        }
+      ]
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "The changelog reads: \"Fixes a stored XSS issue in the Yoast SEO metabox. Thanks Hammad Shamsi for reporting and responsibly disclosing this issue.\""
```

Comment:
Should remove "The changelog reads:" and the discoverer of the vulnerability. CVE policy does not give credit to discoverers in the CVE description. Therefore, "hanks Hammad Shamsi for reporting and responsibly disclosing this issue." must be removed.